BFS_Connected_Components/Q11_most_stones_removed.py

```diff
@@ -0,0 +1,40 @@
+import collections
+
+def build_graph(stones):
+    adj_list = [[] for _ in range(len(stones))]
+    
+    for index1, stone1 in enumerate(stones):
+        for index2, stone2 in enumerate(stones):
+            if index1 == index2:continue
+            if stone1[0] == stone2[0] or stone1[1] == stone2[1]:
```

Comment:
This is not readable. What does `stone1[0]` mean? Separate these out. For example: `stone1_x, stone1_y = stone1`.